### ISSUE TITLE
[Bug][dolphinscheduler-ui] project name can`t contain "http"

### ISSUE THREAD

**Issue by Squidyu** [apache/incubator-dolphinscheduler#4640]
**For better global communication, Please describe it in English. If you feel the description in English is not clear, then you can append description in Chinese(just for Mandarin(CN)), thx! **
**Describe the bug**
Create a new project the name contain "http"，If you enter the project process list, an Error 404 will be reported，and the interface of the new process will also report an Error 404, and the URL path of the front-end splicing is not correct

创建一个名称包含为“http”的新项目。如果进入项目进程列表，会报告404错误，新建流程的接口也会报告404错误，前端拼接的URL路径不正确
**Which version of Dolphin Scheduler:**
 -[all-version]

**Additional context**
The following code may be the cause of the problem：
```
const resolveURL = (url) => {
  if (url.indexOf('http') !== -1) {
    return url
  }
  if (url.charAt(0) !== '/') {
    return `${apiPrefix}/${url.replace(reSlashPrefix, '')}`
  }

  return url
}
```

**Comment by xingchun-chen** [comment]
who can fix it?

**Comment by Ylisen** [comment]
I've fixed it, and I've submitted PR, please check 

[Bug-4640][dolphinscheduler-ui] Fixed a 404 error when the project name contains "http" #4672

**Comment by chengshiwen** [comment]
@CalvinKirs Maybe this issue can be added into `1.3.5-release`

**Comment by CalvinKirs** [comment]
> @chenxingchun @CalvinKirs Maybe this issue can be added into `1.3.5-release`

I noticed that this is not a big problem, so if this PR can be completed before the release, it can be released to 1.3.5, otherwise it will be the next version.
